aiopg/__init__.py

```diff
@@ -28,26 +29,30 @@
 
 
 def _parse_version(ver):
-    RE = (r'^(?P<major>\d+)\.(?P<minor>\d+)\.'
-          '(?P<micro>\d+)((?P<releaselevel>[a-z]+)(?P<serial>\d+)?)?$')
+    RE = (
+        r'^'
+        r'(?P<major>\d+)\.(?P<minor>\d+)\.(?P<micro>\d+)'
+        r'((?P<releaselevel>[a-z]+)(?P<serial>\d+)?)?'
+        r'$'
+    )
     match = re.match(RE, ver)
     try:
         major = int(match.group('major'))
         minor = int(match.group('minor'))
         micro = int(match.group('micro'))
-        levels = {'c': 'candidate',
+        levels = {'rc': 'candidate',
                   'a': 'alpha',
                   'b': 'beta',
                   None: 'final'}
         releaselevel = levels[match.group('releaselevel')]
         serial = int(match.group('serial')) if match.group('serial') else 0
         return VersionInfo(major, minor, micro, releaselevel, serial)
-    except Exception:
-        raise ImportError("Invalid package version {}".format(ver))
+    except Exception as e:
+        raise ImportError("Invalid package version {}".format(ver)) from e
```

Comment:
FTR, in Python 3, exceptions are chained automatically unless you're doing something weird...